### PR TITLE
2021 Dendron community survey results

### PR DESCRIPTION
**Pull request by ScriptAutomate**
Adding first Dendron survey results pages to the Dendron blog

### CODE REVIEW

File: vault/blog.community.2021.new-user-survey.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+## Takeaways
+
+Users are drawn to Dendron for Personal Knowledge Management (PKM), task management, research, and more.
+
+With that in mind, Dendron and the Dendron community have been introducing more features and enhancements that continue to develop:
+
+- [Task Notes](https://wiki.dendron.so/notes/SEASewZSteDK7ry1AshNG/)
+  - [Contribute to the GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358)
+- [Note Traits](https://wiki.dendron.so/notes/bdZhT3nF8Yz3WDzKp7hqh/)
```
Comment (hikchoi) [inline vault/blog.community.2021.new-user-survey.md:138]: Should we link to the quickstart or the API instead of this note? Or add a bit of summary to this note since it's rather empty.

File: vault/blog.community.2021.new-user-survey.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+![Computer sitting on a desk with plants](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-banner.png)
+
+The new user survey was open for several months in 2021, where we were prompting new Dendron users within VS Code itself. We had 2561 respondents take the opportunity to give us feedback.
+
+## What backgrounds do our users have?
+
+![Bar graph displaying different backgrounds of users](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-primary-background.png)
+
+New users were only able to select a single, primary background. Just over half of respondents had a background as **Software Developers** (53%), while the next largest group were primarily **Students** (14%).
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:27]: heads up, this wasn't the case initially (users were able to select multiple). to clear up confusion, just say that users were able to select a background and drop primary distinction

File: vault/blog.community.2021.new-user-survey.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+Approximations of remaining background options:
+
+- **Researcher** (7%)
+- **Manager** (7%)
+- **DevOps** (7%)
+- **Technical Writer** (4%)
+
+The **Other** (8%) group was spread across a variety of results. We were able to generalize the respondents into additional groups.
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:36]: principle: [[get to the point|dendron://private/area.growth.ref.writing-style-guide#get-to-the-point]]
> The **Other** (8%) was spread across the following groups:

File: vault/blog.community.2021.new-user-survey.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+- Systems/Infrastructure/IT/InfoSec Administrators/Engineers (1%)
+  - Solutions Architects
+  - InfoSec / CyberSecurity
+  - Systems Engineers/Administrators
+  - etc.
+- General Hobbyist/Multipotentialite/Polymath (<1%)
+- Data Scientist (<1%)
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:44]: it would be more interesting to see percentage of other (eg. 60% of other had systems it background)

Comment (kevinslin) [reply vault/blog.community.2021.new-user-survey.md:44]: if its too time consuming to calculate, you can leave this in place this time

File: vault/blog.community.2021.new-user-survey.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+- **Research** (47%)
+- **Meeting notes** (45%)
+- **Team knowledge base** (29%)
+- **Publishing** (22%)
+
+A total of 481 (18%) respondents provided additional information under **Other**. Users want to use Dendron for so many things that it was rather difficult to create good categories for them.
+
+> The percentages are in relation to the 481 respondents in **Other**
+
+- Student/Learning/Class/Academia Notes (4%)
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:77]: same comment as above regarding percentages

**Review by kevinslin — CHANGES_REQUESTED** [review]

File: vault/blog.community.2021.new-user-survey.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+## Do users have plans on publishing their notes?
+
+![Bar graph displaying levels of interest in publishing with Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-publishing-interest.png)
+
+New users were only able to select a single response, if they were to respond to this question at all. A majority of users were **Curious** (39%) about publishing, or otherwise already saying **Yes** (36%) to planning publishing. The remaining respondents said **No** (25%) and were not interested in Dendron's publishing capabilities.
+
+## Takeaways
+
+Users are drawn to Dendron for Personal Knowledge Management (PKM), task management, research, and more.
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:132]: [[be specific|dendron://private/area.growth.ref.writing-style-guide#be-specific]]
this is too vague. limit to top 3, eg. "Based on survey results, users are primary drawn to Dendron for {these 3 things}"

File: vault/blog.community.2021.new-user-survey.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+
+With that in mind, Dendron and the Dendron community have been introducing more features and enhancements that continue to develop:
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:134]: this sentence feels like it ends in the middle. 

Also, prefer [[structured docs|dendron://private/area.growth.ref.writing-style-guide#structured-docs]] (add groupings)
```
The Dendron team has been acting on user feedback and shipped some of the following features:
- On PKM
    - feature1
    - feature2
- On Task Management
    ...
```

File: vault/blog.community.2021.new-user-survey.md [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+
+- [Task Notes](https://wiki.dendron.so/notes/SEASewZSteDK7ry1AshNG/)
+  - [Contribute to the GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358)
+- [Note Traits](https://wiki.dendron.so/notes/EQoaBI8A0ZcswKQC3UMpO/)
+- [The Open PKM Catalogue](https://pkm.dendron.so): This work-in-progress is meant to be a reference of all things PKM.
+- Improvements to [importing existing Markdown](https://wiki.dendron.so/notes/f23a6290-2dec-45dc-b616-c218ee53db6b/)
+- Better workflows, such as using [multiple vaults with private and public notes](https://wiki.dendron.so/notes/6DZiBObwhZNYRjnokQ422/)
+
+Though half of respondents said they were developers, the other half of new users came from tech and non-tech backgrounds. This includes many users where Dendron is their introduction to [VS Code](https://code.visualstudio.com/), [git, and GitHub](https://wiki.dendron.so/notes/vsAxYld4zsbD2XjKoh7OP/).
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:143]: Grammar and follow tense of previous sentence. "This includes many users for whom Dendron was their first introduction..."

File: vault/blog.community.2021.new-user-survey.md [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+
+Here are a few areas where we've taken action:
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:145]: Add transitions: 

```
Based on this, we've shipped some of the following features to make it easier for these users to get started with Dendron
```

File: vault/blog.community.2021.new-user-survey.md [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+
+- [Refreshed Getting Started Guide](https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/)
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:147]: [[provide context|dendron://private/area.growth.ref.writing-style-guide#provide-context]]
- add a sentence explaining why. example
```
- [Refreshed Getting Started Guide](https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/): This was a complete redesign of our getting started tutorial. We've added links to all concepts as well as step by step instructions on each option (instead of assuming prior knowledge on the user)
- ...
```

File: vault/blog.community.2021.new-user-survey.md [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+- GitHub Templates for easier, more automated publishing
+  - [Publish with Netlify](https://wiki.dendron.so/notes/yetuum6o9wZi6eVJQBbQb/)
+  - [Publish with GitHub Pages and GitHub Actions](https://wiki.dendron.so/notes/FnK2ws6w1uaS1YzBUY3BR/)
+- Improved `Change Workspace` and `Initialize Workspace` UX
+
+Many of our new users are coming to Dendron as their first tool for personal knowledge management than from any other tool. Even then, the amount of different workflows and tooling that people come from (or are using in parallel) is a massive list.
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:153]: > Many of our new users are coming to Dendron as their first tool for personal knowledge management ~~than from any other tool~~.

Comment (kevinslin) [reply vault/blog.community.2021.new-user-survey.md:153]: > For users coming in with prior experience, there was no single tool that folks were migrating from - instead, we had a massive list of many dozen tools.

File: vault/blog.community.2021.new-user-survey.md [hunk 12]
```diff
@@ -0,0 +1,173 @@
...
+
+We have a [migration guide](https://wiki.dendron.so/notes/f9b4fc21-7613-4c8a-9257-cec4c06b67f9/) that can be referenced for people wanting to import their existing notes into Dendron. With so many tools out there, we are open to [any docs contributions and improvements](https://github.com/dendronhq/dendron-site/blob/master/vault/dendron.guides.migration.md) whenever someone transforms into a dendronite.
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:155]: For folks coming from existing tools,

Comment (kevinslin) [reply vault/blog.community.2021.new-user-survey.md:155]: active style. call to action. "we are looking to the community to help us improve our migration guide"

File: vault/blog.community.2021.new-user-survey.md [hunk 13]
```diff
@@ -0,0 +1,173 @@
...
+
+Within the Dendron Discord, users have mentioned their interest in _interoperability_: they want a single source that they export/import their notes to/from. Our users do this with pods or their own tooling. Different tools co-exist in workflows and can be used concurrently, which further strengthens our belief [that pods are important](https://wiki.dendron.so/notes/66727a39-d0a7-449b-a10d-f6c438185d7f/).
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:157]: [[provide context|dendron://private/area.growth.ref.writing-style-guide#provide-context]]. what are pods? don't just link to it but spend a sentence setting it up

File: vault/blog.community.2021.new-user-survey.md [hunk 14]
```diff
@@ -0,0 +1,173 @@
...
+
+- The Dendron team reflects this. For example, we make use of Airtable every day and will be releasing an Airtable pod in the near future.
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:159]: Its weird that this is a single bullet point. Would rephrase with sentence and example.

"Internally, the Dendron team makes extensive use of pods - for example, all tasks are exported to [airtable](link to pod) so we can utilize alternative views such as kanban and to better query tasks (eg. all bugs that have had no updates in the last 7 days)"

File: vault/blog.community.2021.new-user-survey.md [hunk 15]
```diff
@@ -0,0 +1,173 @@
...
+
+Dendron also stands out compared to other static site generator (SSG) tools due to our UX/DX in managing and taking notes. The quarter of respondents expressing no interest in publishing may also reflect the absence of popular SSGs (Sphinx, Hugo, Jekyll, etc.) in response to tools our users are familiar with.
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:161]: This is introduced out of the blue. Would not introduce new things in conclusion. Next steps:
- add a part about publishing in the body of the blog
- update this part accordingly

File: vault/blog.community.2021.new-user-survey.md [hunk 16]
```diff
@@ -0,0 +1,173 @@
...
+
+We, ourselves, have had a journey of publishing within the Dendron toolchain. We've gone from [Jekyll](https://jekyllrb.com/), to [11ty](https://www.11ty.dev/), and eventually to [Next.js](https://nextjs.org/) (a story for another time!). Using Dendron's built-in publishing provides the best experience, but users can make use of other SSGs with their favorite themes by [exporting Dendron notes](https://wiki.dendron.so/notes/ccab31c4-e9ca-4ee8-b754-f786a3f3be6e/) and tweaking where needed.
+
+## Last words
+
+Again, to all the new users that took our survey: thank you! Without you, we wouldn't have the insight we do into who is trying out Dendron and what features people find most valuable.
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:167]: The survey isn't about us but about the community we serve. Rephrase

We've set aside the majority of this quarter to iterate on the feedback you've given us and are comitted to making Dendron the absolute best tool (and community) for thought for 2022 and beyond!

**Review by kevinslin — CHANGES_REQUESTED** [review]

**Comment by ScriptAutomate** [comment]
I ended up splitting the takeaways section into multiple subsections, since the flow felt a little jarring without their being headers

File: vault/blog.community.2021.new-user-survey.md [hunk 17]
```diff
@@ -0,0 +1,195 @@
...
+
+### PKM and Task Management
+
+Most users are drawn to Dendron for Personal Knowledge Management (PKM) and task management.
+
+The Dendron team has been acting on user feedback and shipped some of the following features:
+
+- On PKM
+  - [The Open PKM Catalogue](https://pkm.dendron.so): This work-in-progress is meant to be a reference of all things PKM.
+  - Improvements to [importing existing Markdown files](https://wiki.dendron.so/notes/f23a6290-2dec-45dc-b616-c218ee53db6b/).
```
[the remark this answers is not in the document]
Comment (kevinslin) [reply vault/blog.community.2021.new-user-survey.md:146]: also, this should be in the format of 
Also, can we have a standard format for these points? Something like the following?

{Feature/Action}: {Description} {How it made things better}

- Adding more [workflow examples](...): This is a growing list of PKM workflows to help users who came from existing tools (or no tool at all) get started with their favorite (or in some cases first) workflows

[the remark this answers is not in the document]
Comment (kevinslin) [reply vault/blog.community.2021.new-user-survey.md:146]: apply the above to all the takeaways

File: vault/blog.community.2021.new-user-survey.md [hunk 18]
```diff
@@ -0,0 +1,195 @@
...
+  - [`Dendron Doctor`](https://wiki.dendron.so/notes/ZeC74FYVECsf9bpyngVMU/) has also had similar improvements, such as [`findBrokenLinks`](https://wiki.dendron.so/notes/ZeC74FYVECsf9bpyngVMU/#findbrokenlinks), to make it easier to see where documentation improvements can be made.
+
+Many of our new users are coming to Dendron as their first tool for personal knowledge management. For users coming in with prior experience, there was no single tool that folks were migrating from. Instead, we had a massive list of previous tools. For those coming from existing tools, we have a [migration guide](https://wiki.dendron.so/notes/f9b4fc21-7613-4c8a-9257-cec4c06b67f9/) that continues to grow.
+
+> We are looking to the community to [help us improve and expand the source documentation](https://github.com/dendronhq/dendron-site/blob/master/vault/dendron.guides.migration.md)!
+
+### Interoperability
+
+Within the Dendron Discord, users have mentioned their interest in _interoperability_: they want a single source that they export/import their notes to/from. Different tools co-exist in workflows and can be used concurrently.
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:172]: > Different tools co-exist in workflows and can be used concurrently.

I don't understand this sentence

File: vault/blog.community.2021.new-user-survey.md [hunk 19]
```diff
@@ -0,0 +1,195 @@
...
+
+Our users continue to integrate information managed by Dendron with **pods** or their own tooling. In Dendron, [pods](https://wiki.dendron.so/notes/66727a39-d0a7-449b-a10d-f6c438185d7f/) refer to any tool or location that contains information and how Dendron manages that information: [Markdown files](https://wiki.dendron.so/notes/13c4a608-0a32-4c62-9c7e-2b7f9f2d18bf/), [GitHub issues](https://wiki.dendron.so/notes/PuDUEyUAPmvpBvaFWHDOS/), [Notion](https://wiki.dendron.so/notes/eOw3TC9KgdbBvhyU7pyGl/), and social media (Twitter, Facebook, etc.) are pod examples.
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:174]: This description is confusing. Pods are a set of plugins that help users synchronize their notes between Dendron and other tools

File: vault/blog.community.2021.new-user-survey.md [hunk 20]
```diff
@@ -0,0 +1,195 @@
...
+
+Internally, the Dendron team makes extensive use of pods. For example, all tasks are [exported with the Airtable Pod](https://wiki.dendron.so/notes/c8ljkYY3eVkkmTNumc0C7/) so we can utilize alternative views such as Kanban and better task querying (eg. all bugs that have had no updates in the last 7 days).
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:176]: this would benefit from a screenshot

File: vault/blog.community.2021.new-user-survey.md [hunk 21]
```diff
@@ -0,0 +1,195 @@
...
+
+### Publishing
+
+Dendron stands out compared to other SSGs due to our UX/DX in _creating and managing_ notes, in addition to the actual _publishing_ of notes. For the 75% of users curious about or interested in publishing: we maintain GitHub Templates for easier, more automated publishing on popular platforms:
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:180]: what is SSG (I don't think we define it). We also don't talk about it. since this is a takeaway, we should start with an insight. what have we learned from users about publishing? from that, how does dendron publishing help them?

**Review by kevinslin — CHANGES_REQUESTED** [review]

**Comment by ScriptAutomate** [comment]
I've stripped out the takeaways/feedback section to use in a later retrospective post, based on what Dendron has worked on delivering to address new user feedback.

File: vault/blog.community.2021.new-user-survey.md [hunk 22]
```diff
@@ -0,0 +1,140 @@
...
+- Typora (3%)
+- Physical notes (2%)
+
+## Do users have plans on publishing their notes?
+
+![Bar graph displaying levels of interest in publishing with Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-publishing-interest.png)
+
+New users were only able to select a single response, if they were to respond to this question at all. A majority of users were **Curious** (39%) about publishing, or otherwise already saying **Yes** (36%) to planning publishing. The remaining respondents said **No** (25%) and were not interested.
+
+## Last words
```
Comment (kevinslin) [inline vault/blog.community.2021.new-user-survey.md:132]: change to

### What's Next

We've set aside the majority of this quarter (and the last) to iterate on the feedback you've given us (specific details to come in a future blog post).
We are committed to making Dendron the absolute best tool (and community) for thought in 2022 and beyond!
We welcome you to be part of our journey 🌱

**Review by kevinslin — CHANGES_REQUESTED** [review]

**Review by kevinslin — APPROVED** [review]
